stable/node-local-dns/README.md

```diff
@@ -12,37 +12,11 @@ Further documentation is [here](https://kubernetes.io/docs/tasks/administer-clus
 
 This helm chart works for both kube-proxy setups (iptables or ipvs).
 
-## How to install this chart
```

Comment:
This stuff should not be removed. Are you running `docker run --rm -v "$(git rev-parse --show-toplevel):/helm-docs" jnorwood/helm-docs:v1.11.3 --template-files ./ci/README.md.gotmpl`?

Reply:
Can you try running it from repo root @SadmiB ?

Reply:
Nice, regenerated from root.